cmd/main.go

```diff
@@ -128,14 +86,15 @@ func realMain(ctx context.Context) error {
 				return
 			case <-ticker.C:
 				logger.DebugContext(ctx, "Updating metrics definitions.")
-				// Error logged by db.
-				_ = db.Update(ctx, dbUpdateParams)
+				if err = db.Update(ctx, dbUpdateParams); err != nil {
+					logger.WarnContext(ctx, "Error updating metrics definitions, will use cached definition if available.", "err", err.Error())
+				}
 			}
 		}
 	}()
 
 	mux := http.NewServeMux()
-	mux.Handle("POST /sendMetrics", handleMetric(h, db))
+	mux.Handle("POST /sendMetrics", server.HandleMetric(h, db))
```

Comment:
Wasn't aware we can specify POST this way

Reply:
It's new in https://go.dev/blog/go1.22

Reply:
https://pkg.go.dev/net/http#hdr-Patterns

Reply:
Thanks!